client/index.css

```diff
@@ -199,3 +199,115 @@ ul {
 .chat ul li.typing div {
     animation: flash 2s infinite;
 }
+#settings {
+    position: absolute;
+    top: 40px;
+    left: 0;
+    right: 0;
+    width: 345px;
+    height: auto;
+    margin: auto;
+}
+#settings #icon-username {
+    margin: 18px 0 0 100px;
+    overflow: auto;
+}
+#icon-username h2 {
+    margin-left: 120px;
+}
+#settings .icon {
+    width: 100px;
+    height: 150px;
+    border: 1px solid black;
+    float: left;
+    margin-top: -45px;
+}
+.icon img {
+    height: 100px;
+    width: 98px;
+}
+.icon span {
+    padding: 4px 10px;
+    border-top: 1px solid black;
+    cursor: pointer;
+    display: block;
+}
+#settings #form {
+    position: absolute;
+    top: 225px;
+    width: 400px;
+}
+#form select {
+    width: 174px;
+    display: inline-block;
+    padding: 0 0 0 9px;
+}
+#form select.day,
+#form select.month {
+    width: 50px;
+}
+#form select.year {
+    width: 64px;
+}
+#form select.month,
+#form select.year {
+    margin-left: 5px;
+}
+#form label {
+    width: 150px;
+    margin: 0px 20px 30px 0px;
+    text-align: right;
+}
+#form input {
+    width: 173px;
+    display: inline-block;
+}
+#form .button {
+    position: relative;
+    left: 150px;
+    color: white;
+    background-color: #0084ff;
+    border: 1px solid #0084ff;
+    border-radius: 5px;
+    padding: 10px 20px;
+    text-decoration: none;
+}
+.top .topbar {
+    position: absolute;
+    top: 10%;
+    right: 22px;
+}
+.topbar img.avatar {
+    border-radius: 50px;
+    width: 30px;
+    height: 30px;
+    cursor: pointer;
+}
+.topbar button {
+    background-color: transparent;
+    border-color: transparent;
+    width: auto;
+    height: auto;
+    padding: 0;
+}
+.topbar button:hover, .topbar button:active {
+    background-color: transparent;
+    border-color: transparent;
+    outline: none !important;
```

Comment:
Avoid `!important`. See if you can have the same result using selector specificity.